fractal-theme/assets/scss/src/_changelog.scss

```diff
@@ -0,0 +1,68 @@
+.changelog {}
+
+.changelog-item {

```

Comment:
Why not using SCSS to shorten down selectors as we usally do (now when we are in fact using scss)
Also, why not use BEM?

`&__item` would be a better solution.

Reply:
This applies for the entire file.

Reply:
I chose not to use BEM because the fractal theme is not built using BEM, so it feels "out of place" - and because I didn't use BEM, I didn't want to use the &-pattern, as I think it looks wrong, when used with only a single dash.

Reply:
makes sense